pkg/urbit/include/vere/vere.h

```diff
@@ -1,5 +1,9 @@
-/* include/vere/vere.h
+/* pkg/urbit/include/vere/vere.h
+**
+** This file is in the public domain.
 */
+#ifndef URBIT_VERE_VERE_H
+#define URBIT_VERE_VERE_H
```

Comment:
This should just be `U3_VERE_H`.

Reply:
On a similar note, should the header guard in [`serf.h`](https://github.com/urbit/urbit/tree/peter/header-guards/pkg/urbit/include/vere/serf.h) be `U3_VERE_SERF_H` or `U3_SERF_H`?

Reply:
Either one is fine, I'm gonna remove that file soon.

Reply:
Went with `U3_VERE_SERF_H`.